README.md

```diff
@@ -139,8 +139,8 @@ the implementation.
 Languages supported by the Bytecode Alliance:
 
 * **[Rust]** - the [`wasmtime` crate]
-* **[C]** - the [`wasm.h`, `wasi.h`, and `wasmtime.h` headers][c-headers], [CMake](crates/c-api/CMakeLists.txt) or [`wasmtime` Conan package]
-* **C++** - the [`wasmtime.hh` header][c-headers] or the [`wasmtime-cpp` Conan package]
+* **[C]** - the [`wasm.h`, `wasi.h`, and `wasmtime.h` headers][c-headers]
+* **C++** - the [`wasmtime.hh` header][c-headers]
```

Comment:
What is the motivation behind removing the cmake and conan stuff? I have no idea if it is worth listing those things, as I do very little C/C++ stuff, but it seems like something we should change with intention.